apiary.apib

```diff
@@ -625,6 +625,20 @@ Get details for a single user.
             + function: `Sales` (string)
             + time_zone: `Europe/Brussels` (string)
 
+
+### users.updateInternalHourlyCost [POST /users.updateInternalHourlyCost]
+
+Update details for a single user.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `b5094b3f-bb7a-0391-b01b-e709773f3509` (string)
+        + internal_hourly_cost (object, nullable)
+            + value (Money)
```

Comment:
Not sure it's worth having value as a separate field inside internal_hourly_cost anymore. It was useful when the endpoint was more generic (i.e. updateRates). But now that it refers specifically to the internal hourly cost, I guess we don't need nested fields. You can easily just add a `from` and a `to` at the base level, along `internal_hourly_cost`

Reply:
Precisely. Now that `value` is a bit over-engineered.

Reply:
indeed. updated.